src/components/PaginationInfo/PaginationInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface Props {
+  total: number;
+  currentPage: number;
+  firstItem: number;
+  lastItem: number;
+}
+
+export const PaginationInfo: React.FC<Props> = ({
+  total,
+  currentPage,
+  firstItem,
+  lastItem,
+}) => {
+  return (
+    <p className="lead" data-cy="info">
+      {`Page ${currentPage} (items ${firstItem} - ${lastItem} of ${total})`}
```

Comment:
Ensure that the text content respects the potential edge cases. For example, when there are no items to display, the message should reflect that there are no items instead of showing 'items 0 - 0 of 0'. It would be more user-friendly to check if `total` is 0 and adjust the message accordingly.